boilerplates/aws-lambda-serverless/files/$.env.ts

```diff
@@ -0,0 +1,34 @@
+import { appendToEnv, type TransformerProps } from "@batijs/core";
+
+export default async function getServerlessEnv(props: TransformerProps) {
+  if (!props.meta.BATI.has("sentry")) return undefined;
+  const sentryOrg = process.env.TEST_SENTRY_ORG;
+  const sentryProject = process.env.TEST_SENTRY_PROJECT;
+  const sentryAuthToken = process.env.TEST_SENTRY_AUTH_TOKEN;
+  const sentryDNS = process.env.TEST_SENTRY_DSN;
+
+  let envContent = await props.readfile?.();
+
+  envContent = appendToEnv(envContent, "SENTRY_ORG", sentryOrg ?? "", "Sentry Org. Used for Upload of Source Maps");
+  envContent = appendToEnv(
+    envContent,
+    "SENTRY_PROJECT",
+    sentryProject ?? "",
+    "Sentry Project Slug. Used for Upload of Source Maps",
+  );
+  envContent = appendToEnv(
+    envContent,
+    "SENTRY_AUTH_TOKEN",
+    sentryAuthToken ?? "",
+    "Sentry Auth Token. Used for Upload of Source Maps",
+  );
+  envContent = appendToEnv(envContent, "SENTRY_DSN", sentryDNS ?? "", "Sentry DNS. Used for Error Reporting");
+  envContent = appendToEnv(
+    envContent,
+    "SENTRY_PROFILER_BINARY_PATH",
+    process.env.SENTRY_PROFILER_BINARY_PATH ?? "",
+    "SENTRY_PROFILER_BINARY_PATH. Needed for location of profiler binary",
+  );
+
+  return envContent;
+}
```

Comment:
`sentry` integration should be moved in its own boilerplate.
You should also enable the Sentry feature in `features.ts`.

Reply:
> `sentry` integration should be moved in its own boilerplate. You should also enable the Sentry feature in `features.ts`.

I have activated Sentry in `features.ts` and created a boilerplate. But the only thing that would live there might be an `$.env.ts` with `SENTRY_DSN`.

Sentry a lib per front and backend plattform and sometimes frameworks - here are some examples:
* REACT - `@sentry/react`
* Astro (React framework) - `@sentry/astro` 
* AWS Lambda / Javascript - `@sentry/aws-serverless`
* Node - `@sentry/node`
* Hono -`@hono/sentry`
* Cloudflare - https://developers.cloudflare.com/workers/observability/sentry-integration/
* Vercel - https://docs.sentry.io/organization/integrations/deployment/vercel/

It is even more complicated when you combine some of the above like in my case `hono` with `AWS Lambda`.
I could use `@hono/sentry` but then I will miss the special features of `@sentry/aws-serverless` which is also tracking timeouts and other AWS Lambda specific information. This is why I implemented the hono error handler to use the sentry instance from `@sentry/aws-serverless`: 
https://github.com/vikejs/bati/blob/63c647cd248c886a74de5f957b87550045272ef9/boilerplates/hono/files/hono-entry.ts#L37C1-L49

Reply:
> I have activated Sentry in features.ts and created a boilerplate. But the only thing that would live there might be an $.env.ts with SENTRY_DSN.

That's a good enough reason to create a new boilerplate. Not everything has to live within it, but if it conceptually makes sense for somes files because they can be reused by other bati components, it should be move to a separate boilerplate.